test/amplitude.js

```diff
@@ -12,6 +13,7 @@ import Revenue from '../src/revenue.js';
 // maintain for testing backwards compatability
 describe('Amplitude', function() {
   var apiKey = '000000';
+  const cookieName = 'amp' + apiKey.slice(0,6);
```

Comment:
hmm why not just make the new cookie `amp` + the existing `keySuffix`?

Reply:
I'm try to save as many bytes as sanely possible. Cookies don't give you much to work with.